src/VisualStudio/Core/Def/ValueTracking/ValueTrackingToolWindow.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Windows;
+using System.Windows.Controls;
+using Microsoft.CodeAnalysis;
+using Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.VisualStudio.LanguageServices.ValueTracking
+{
+    [Guid(Guids.ValueTrackingToolWindowIdString)]
+    internal class ValueTrackingToolWindow : ToolWindowPane
+    {
+        public static ValueTrackingToolWindow? Instance { get; set; }
+        private readonly ValueTrackingTreeViewModel _viewModel;
+
+        // Needed for VSSDK003
+        // See https://github.com/Microsoft/VSSDK-Analyzers/blob/main/doc/VSSDK003.md
```

Comment:
We should make sure that we eventually change the tool window to be async. Synchronous tool window construction is surprisingly expensive.

Reply:
Yes, plus async tool windows are not difficult to implement.

Reply:
I _think_ I fixed this. I was close to having it, didn't realize I should just not block on window show. I followed https://github.com/madskristensen/AsyncToolWindowSample/